### PR TITLE
Update Vietnamese translation

### PR DESCRIPTION
**Pull request by tuan041**
Update Vietnamese translation

### CODE REVIEW

**Comment by tuan041** [comment]
Can someone fix this for me?
Pls

**Comment by LagradOst** [comment]
`ERROR:/home/runner/work/CloudStream-3/CloudStream-3/app/src/main/res/values-vi/strings.xml: Resource and asset merger: Found item String/eigengraumode_settings more than one time`

File: app/src/main/res/values-vi/strings.xml [hunk 1]
```diff
@@ -113,6 +156,7 @@
     <string name="chromecast_subtitles_settings">Phụ đề Chromecast</string>
     <string name="chromecast_subtitles_settings_des">Cài đặt phụ đề Chromecast</string>
 	
+    <string name="eigengraumode_settings">Chế độ Eigengravy</string>
     <string name="eigengraumode_settings">Điều chỉnh tốc độ khi xem phim</string>
```
Comment (Blatzar) [inline app/src/main/res/values-vi/strings.xml:160]: This is the duplicate key causing the error.

Comment (tuan041) [reply app/src/main/res/values-vi/strings.xml]: Thanks

**Comment by LagradOst** [comment]
LGTM, ready to merge? 👍

**Comment by tuan041** [comment]
Yep

**Pull request merged**